Tests/FoundationEssentialsTests/DateTests.swift

```diff
@@ -90,15 +98,18 @@ final class DateTests : XCTestCase {
         XCTAssertEqual("1970-01-01 00:00:00 +0000", date.description)
     }
 
-    func testDescriptionDistantPast() {
+    func testDescriptionDistantPast() throws {
+        try XCTSkipIf(libcIsUCRT(), "ucrt does not support distant past")
```

Comment:
I think I'd prefer a style here where we simply put the `if os(Windows)` in these two places directly instead of behind the function. The reasoning is that it's a lot more clear when looking at individual tests, and I don't think we expect to have many Date tests skipped for Windows.

Reply:
These two are the ones that I expect to have this temporarily for. However, that does cause a warning, which is why I was preferring to do this. If you feel strongly, I don't mind doing something like:

```swift
#if os(Windows)
try XCTSkipIf(true, "...")
#else
...
#endif
```

Reply:
The style that we've used in other tests like `FileManagerTests` is

```swift
func testFoo() throws {
    #if os(Windows)
    throw XCTSkip("This test is not applicable because ...")
    #else
    /* ... test ... */
    #endif
}
```

Maybe we could use that here similar to your suggest above but a little more concise? I don't think that should produce any warnings